utils.py

```diff
@@ -0,0 +1,9 @@
+def separate_callback_data(data):
+    return data.split(";")
+
```

Comment:
If this is the same for `telegramcalendar.py` and `telegramjcalendar.py`, it is ok if it is in this `utils.py` module. If it is only used by `telegramjcalendar.py` let's move it to that module

Reply:
Yes, it's the smae for both calendars. I remove this function from `telegramcalendar.py` and replcae with the one in `utils.py`.